meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -199,33 +196,31 @@ <h2>{{ event.name }}</h2>
                     </ul>
                 </div>
             </div>
-        </div>
         {% else %} <!-- if just one module and no phase view to dispatch to -->
         {% include "a4modules/includes/module_detail_phase.html" %}
         {% endif %}
         <!-- these blocks are only filled when in phase view -->
-        <div class="container">
             <div class="offset-lg-3 col-lg-6">
                 {% block project_action %}{% endblock %}
             </div>
         </div>
```

Comment:
The closing div tag also needs to go

Reply:
@fuzzylogic2000 that's matching the tag at 175 as i just removed all the additional containers